docs/Getting-Started.md

```diff
@@ -291,7 +291,7 @@ Read [Validation and Serialization](Validation-and-Serialization.md) to learn mo
 <a name="extend-server"></a>
 ### Extend your server
 Fastify is built to be extremely extensible and minimal, we believe that a bare bones framework is all that is necessary to make great applications possible.<br>
-In other words, Fastify is not a "batteries included" framework, and relies on an amazing [ecosystem](Ecosystem.md)!
+In other words, Fastify is not a "batteries included" framework, and relies on an amazing [ecosystem](https://www.fastify.io/ecosystem/)!
```

Comment:
I think a better fix would be to fix the website builder here

https://github.com/fastify/website/blob/06409ed51c972bc4d3682e00419f0807487b4cb5/src/scripts/processReleases.js#L144-L149

So the user that navitages in the site, will remains in the site and if the dev navigates through github will stay here

Reply:
Yeah this totally make sense